components/molecules/Menu.js

```diff
@@ -31,22 +22,45 @@ export function Menu(props) {
       <h3 className="sr-only" id="mainSiteNav">
         Menu
       </h3>
-      <button
-        id="menuButton"
-        onClick={onMenuClick}
-        className="text-h4 text-canada-footer-font focus:outline-none focus:ring-2 focus:ring-black mb-4 py-1"
-        aria-haspopup="true"
-        aria-expanded="false"
-        aria-controls="menuDropdown"
-        data-testid="menuButton"
-      >
-        <span className="inline-block align-middle icon-menu" />
-        <span className="inline-block align-middle pl-3 font-body text-p leading-none">
-          {props.menuButtonTitle}
-        </span>
-      </button>
+      <div className="flex justify-between">
+        <button
+          id="menuButton"
+          onClick={() => setShowMenu(!showMenu)}
+          className="text-h4 text-canada-footer-font focus:outline-none focus:ring-2 focus:ring-black mb-4 py-1"
+          aria-haspopup="true"
+          aria-expanded={showMenu}
+          aria-controls="menuDropdown"
+          data-testid="menuButton"
+        >
+          <span className="inline-block align-middle icon-menu" />
+          <span className="inline-block align-middle pl-3 font-body text-p leading-none">
+            {props.menuButtonTitle}
+          </span>
+        </button>
 
-      <ul id="menuDropdown" className="menuDropdown" role="menu">
+        <button
+          id="menuClose"
+          onClick={() => setShowMenu(!showMenu)}
+          className={`${
+            showMenu ? "" : "hidden"
+          } sr-only font-body flex text-canada-footer-font outline-none focus:not-sr-only focus:outline-black-solid lg:invisible`}
+          aria-expanded={showMenu}
+          aria-controls="menuDropdown"
+          aria-label="Close"
+          data-testid="menuCloseButton"
+        >
+          <span id="close" className="text-h2">
+            &times;
```

Comment:
I think we got some feedback in the feedback component a11y audit that says we should avoid using this format for the close button span value. I think it's because the screen reader will actually read out "times close button" when it is looking at the close menu button. More detail on their feedback is in #6 here: https://docs.google.com/spreadsheets/d/1HY1tHL6YTsapS0jCcD5w6hm9uIfFCHIk8AHC4ITY1wg/edit#gid=0